### ISSUE TITLE
Add read.amazon.com

### ISSUE THREAD

**Issue by EusebiuOprinoiu** [AdguardTeam/HttpsExclusions#560]
Please add `read.amazon.com` to the list of subdomains excluded from filtering.
This subdomain is used for both the Web Kindle Reader and to show sample previews for ebooks on Amazon.
Without the exclusion, the sample previews do not load. A normal filter exclusion doesn't work.

**Comment by Alex-302** [comment]
Hi
Which AdGuard version, filters and browser?

**Comment by EusebiuOprinoiu** [comment]
Hello, Alex!

I'm using the latest version of AdGuard (17.16.0) with the filters below:
- AdGuard Base Filter
- AdGuard URL Tracking Filter
- AdGuard Tracking Protection Filter
- AdGuard Social Media
- Adguard Annoyances

The issue happens on all browsers, even when all filters are disabled.
The only way to fix this is to add an HTTPS exclusion.

**Comment by Alex-302** [comment]
HTTPS exclusion is not required here. Will fix.

**Comment by EusebiuOprinoiu** [comment]
Thank you very much. I appreciate it!

**Comment by contribucious** [comment]
@Alex-302 👋

**Important remark regarding your current `read.amazon.*` stealth fix _(using a popular dev ebook as an example here)_:**

✔️
<sup>
https://www.amazon.com/dp/B088P9Q6BB -> OK because `read.amazon.com` will be used.
https://www.amazon.co.uk/dp/B088P9Q6BB -> OK because `read.amazon.co.uk` will be used.
</sup>

✖️
<sup>
https://www.amazon.fr/dp/B088P9Q6BB -> NOK because `lire.amazon.fr` will be used.
https://www.amazon.it/dp/B088P9Q6BB -> NOK because `leggi.amazon.it` will be used.
https://www.amazon.de/dp/B088P9Q6BB -> NOK because `lesen.amazon.de` will be used.
…
</sup>

Basically, the word "read" is translated.
So I think the only way is to find a common part, like the path one (`/sample/`):
```adblock
@@||amazon.*/sample/$stealth
```

**Comment by Alex-302** [comment]
But I guess not only sample is broken.

**Comment by Alex-302** [comment]
No, full books are not broken.

Checked again, it seems `$content,extension,elemhide,jsinject` is not required. Replaced.

**Comment by contribucious** [comment]
👍

However, I guess you meant `!+ PLATFORM(windows, mac, android)` instead of `!+ NOT_PLATFORM(windows, mac, android)`?

###### 💻 Env.: _AdGuard for Windows 7.17.0 nightly 9_.

**Comment by Alex-302** [comment]
Yes, thanks.
